deploy-edpm.yml

```diff
@@ -35,6 +35,17 @@
 - name: Import admin setup related playbook
   ansible.builtin.import_playbook: ci_framework/playbooks/07-admin-setup.yml
 
+  # Note(Chandan): It is a know issue with the dev-preview nova-operator that
+  # it runs the libvirt and nova-compute deployment twice but reports ready 
+  # status after the first run. So with this sleep the job waits for the 
+  # deployment to really stabilize before start running the tempest tests.
+- name: Sleep for 5 mins to finish th EDPM deployment
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  tasks:
+    - name: Add a sleep for 5 mins
+      ansible.builtin.pause:
```

Comment:
no other way than a hard-coded pause?

Reply:
Nope, This will be probably fixed in the main branch. Once Dev preview is released. We can revert this patch.

Reply:
And there is no other way? There is no log trace that points that is ready? No service exposed? Nothing?

Reply:
If there is nothing to poll for, I'd approve this.

Reply:
if you want to complicate things then you can try to watch for running deployment jobs for 5 minutes after the deployment was ready. But as it also has the 5 minutes magic number I don't think it worth the complexity.